lib/rspec/mocks/object_reference.rb

```diff
@@ -90,25 +113,41 @@ def target
     # Represents a reference to the object named (via a constant lookup)
     # by the string.
     #
-    # @private
+    # @param string constant name
     class NamedObjectReference
       def initialize(const_name)
         @const_name = const_name
       end
 
+      # The definition state of the object.
```

Comment:
I think "definition state" is a pretty vague, abstract description that might not mean anything to users.  (Plus it is using the same root -- define -- as the method name, so it's a bit like a dictionary defining a word with a different form of the same word).  What do you think of this instead?

``` ruby
# Returns true if the named constant is defined, false otherwise.
```